client-app/pages/product.vue

```diff
@@ -81,6 +92,20 @@ const seoKeywords = computed(() => product.value?.seoInfo?.metaKeywords);
 const seoImageUrl = computed(() => product.value?.imgSrc);
 const hasVariations = computed(() => productHasVariations(product.value));
 
+const productInfoSection = computed(
+  () => template.value?.content.filter((item: PageContent) => item.type === "product-info")[0],
+);
+
+const productVariationsBlock = computed(
+  () =>
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
why any?

Reply:
Because of this https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/dev/client-app/shared/static-content/types/index.ts#L10